src/features/applicationView/offerCard/sendInvoiceForm/__tests__/SendInvoiceForm.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+
+import SendInvoiceForm, { SendInvoiceFormProps } from '../SendInvoiceForm';
+
+const mockProps: SendInvoiceFormProps = {
+  email: 'test@example.com',
+  isSubmitting: false,
+  onCancel: jest.fn(),
+  onSubmit: jest.fn(),
+};
+
+const realDateNow = Date.now.bind(global.Date);
+
+beforeAll(() => {
+  global.Date.now = jest.fn(() => new Date('2020-09-23T00:00:00.000Z').valueOf());
+});
+
+afterAll(() => {
+  global.Date.now = realDateNow;
+});
+
+describe('SendInvoiceForm', () => {
+  const getWrapper = (props?: Partial<SendInvoiceFormProps>) => mount(<SendInvoiceForm {...mockProps} {...props} />);
+
+  it('renders normally', () => {
+    const wrapper = getWrapper();
+    expect(wrapper.render()).toMatchSnapshot();
+  });
+
+  it('shows an error if email is missing', () => {
+    const wrapper = getWrapper({ email: null });
+    expect(wrapper.find('Text.missingEmail')).toMatchInlineSnapshot(`
```

Comment:
Text.missingEmail is an implementation detail. This is also good example where testing library is better